magenta/interfaces/midi/midi_hub.py

```diff
@@ -140,6 +140,7 @@ class Metronome(threading.Thread):
   Args:
     outport: The Mido port for sending messages.
     qpm: The integer quarters per minute to signal on.
+    qpb: The number of quarter notes per bar.
```

Comment:
On second thought, let's have this after start_time since that is more likely to be non-default